packages/babel-plugin-proposal-optional-chaining/src/index.js

```diff
@@ -50,9 +59,10 @@ export default declare((api, options) => {
 
           let ref;
           let check;
-          if (loose && isCall) {
+          if (loose && isCall && isSimpleMemberExpression(node.callee)) {
```

Comment:
I think this can be as simple as `!t.isCallExpression(chain)` and still get the same behavior.

Note, `chain` is just a local variable for `node.callee`.